source/OptimaJet.DWKit.StarterApplication/Services/Contracts/IHangfireWrapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Hangfire;
+
+namespace OptimaJet.DWKit.StarterApplication.Services
+{
+    public interface IHangfireWrapper
```

Comment:
I would be good to have comments on why using IHangfireWrapper instead of injecting IBackgroundJobClient.

http://www.jondjones.com/c-sharp-bootcamp/tdd/misc-unit-testing-snippets/how-to-unit-test-your-hangfire-tasks